barretenberg/cpp/src/barretenberg/dsl/acir_proofs/c_bind.cpp

```diff
@@ -249,6 +250,22 @@ WASM_EXPORT void acir_write_vk_ultra_honk(uint8_t const* acir_vec, uint8_t** out
     *out = to_heap_buffer(to_buffer(vk));
 }
 
+WASM_EXPORT void honk_solidity_verifier(uint8_t const* acir_vec, uint8_t** out)
```

Comment:
this and the other functions should be more aptly named, like get_honk_solidity_verifier_vk

Reply:
agree, seems like there's no standard for the naming... I'd say this should be done in a different PR? trying not to scope creep too much...